### ISSUE TITLE
Validate JS injection rules body

### ISSUE THREAD

**Issue by jellizaveta** [AdguardTeam/AGLint#256]
Validate JS injection rules body with [acorn](https://www.npmjs.com/package/acorn).

So, from

```adblock
example.org#%#window.__gaq = undefined;
```

validate
```js
window.__gaq = undefined;
```

If this part is syntactically incorrect, acorn will simply throw an error.

ref: https://github.com/AdguardTeam/AGLint/issues/13#issuecomment-1420559391
